src/CMakeLists.txt

```diff
@@ -46,8 +46,11 @@ set(BA_SOURCES
     ${CMAKE_CURRENT_SOURCE_DIR}/base_alloc/base_alloc_linear.c
     ${CMAKE_CURRENT_SOURCE_DIR}/base_alloc/base_alloc_global.c)
 
```

Comment:
pls add info in the commit msg where was the ctl taken from...

Reply:
Done